docs/updating/8-0.md

```diff
@@ -74,6 +74,26 @@ npm install @ionic/core@next
 
 The following changes are not required to update to Ionic 8 as your application will continue to work. However, we recommend making the following changes to ensure you can use the new features in Ionic 8.
 
+### Light Theme
+
+Previous versions shipped a set of default color variables for the light theme:
+
+```css
+/** Ionic CSS Variables **/
+:root {
+  /** primary **/
+  --ion-color-primary: #3880ff;
+  --ion-color-primary-rgb: 56, 128, 255;
+  --ion-color-primary-contrast: #ffffff;
+  --ion-color-primary-contrast-rgb: 255, 255, 255;
+  ...
+}
+```
+
+If you are not using a custom color scheme, these color variables should all be removed. In Ionic Framework version 8, the default colors have been updated to improve accessibility. The new palette is also automatically included with all Ionic apps. Removing the old color variables ensures that the latest palette is not overwritten.
```

Comment:
```suggestion
In Ionic Framework version 8, the default color palette is included as long as `core.css` is imported. Removing the old color variables ensures that the latest palette is not overwritten.

Developers who are customizing this color palette can continue to keep the custom variables values, but any of the variables that use the default values should be removed.
```

Optional: I changed a bit of the wording. Mentioning the new a11y improvements isn't required to make this change, so I opted to get rid of it. I also wanted to call out the relationship between the light theme and the `core.css` file.

Reply:
Works for me! I was a bit unsure of the wording; agree that this is much cleaner.